pkg/testing/fetcher_artifact.go

```diff
@@ -84,7 +93,7 @@ func (f *artifactFetcher) Fetch(ctx context.Context, operatingSystem string, arc
 	}
 
 	// this remote path cannot have the build metadata in it
-	srcPath := fmt.Sprintf("elastic-agent-%s-%s", ver.VersionWithPrerelease(), suffix)
+	srcPath := fmt.Sprintf("elastic-agent-%s-%s-%s", prefix, ver.VersionWithPrerelease(), suffix)
```

Comment:
TODO: verify that this is the correct structure once FIPS-compliant Agent artifacts are available.